control_panel_api/tests/test_permissions.py

```diff
@@ -426,10 +426,10 @@ def test_when_not_authenticated_responds_403(self):
         response = self.client.get('/k8s/something')
         self.assertEqual(HTTP_403_FORBIDDEN, response.status_code)
 
-    @patch('kubernetes.client.configuration')
-    @patch('kubernetes.config.load_incluster_config')
+    @patch('kubernetes.client.configuration', MagicMock())
```

Comment:
nice 👍

Reply:
No, default causes a param to be injected in test methods. Specifying here prevents that.